python/ray/autoscaler/command_runner.py

```diff
@@ -717,8 +721,14 @@ def run_init(self, *, as_head, file_mounts):
 
             self.run("docker pull {}".format(image), run_env="host")
 
+        # Don't mount bootstrap files because of docker mount inode behavior.
+        # (Changes to host file are not reflected in container).
+        cleaned_file_mounts = file_mounts.copy()
+        for mnt in BOOTSTRAP_MOUNTS:
+            cleaned_file_mounts.pop(mnt, None)
```

Comment:
hmm i'm a little confused here actually. it looks like you're trying to _not_ include the bootstrap config in the docker container? how is the autoscaler supposed to run ?

Reply:
It seems to me that this is removing the bootstrap key from the mounts, and the PR instead uses rsync and cp to get them in, right?

Reply:
oh i see, i missed that part at the end. i suppose my higher level question is why do we have to special case these at all?

Reply:
Correct @pcmoritz